### DISCUSSION TITLE
neper generate a   2-scale Voronoi model

### DISCUSSION THREAD

**Discussion by neperchh** [neperfepx/neper#660] [Q&A] [answered]
Dear techers, I want generate a  2-scale Voronoi model .Firstly I want generate a model  with specific seed coordinates，secondly add parallel blocks on his basis.    **neper -T -n  500::from_morpho  -morphooptiini "coo:file(coo),weight:0" ::-morpho"lamellar(w=0.1)"**        the code was written by me，but it is wrong. Please help me,thanks.

**Comment by rquey** [comment, marked as answer]
```
 neper -T -n 500::from_morpho -morphooptiini "coo:file(coo),weight:0" -morpho"voronoi::lamellar(w=0.1)" 
```

**Reply by neperchh** [reply]
Thank you very much for your reply ,which is very useful to me.
